passes/elliminate_undefined.hh

```diff
@@ -0,0 +1,21 @@
+#ifndef ELLIMINATE_UNDEFINED_HPP_WSIRHVJZ
+#define ELLIMINATE_UNDEFINED_HPP_WSIRHVJZ
```

Comment:
I appreciate your creativity, but can you use something more common?
`H_ELLIMINATE_UNDEFINED_H`

Reply:
Of course, sorry. I have a macro that does this automatically.